### PR TITLE
Update workshop content for OCP4 service mesh

### PR DESCRIPTION
**Pull request by ajacocks**

### CODE REVIEW

File: content/workshops/openshift_4_101_dynatrace/fin.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Fin.
-workshops: openshift_4_101
+workshops: openshift_4_101_dynatrace
 workshop_weight: 20
```
Comment (bsalisbury-emergent360) [inline content/workshops/openshift_4_101_dynatrace/fin.md:4]: Should be changed to "workshop_weight: 23". Otherwise "fin" shows up between lab9 and lab 10

Comment (ajacocks) [reply content/workshops/openshift_4_101_dynatrace/fin.md]: Fixed; thanks!

**Review by jvanmeter — APPROVED** [review]
Approved

**Comment by jvanmeter** [comment]
Merging

**Pull request merged**